R/lazyframe-frame.R

```diff
@@ -1021,7 +1021,7 @@ lazyframe__drop_nulls <- function(subset = NULL) {
   })
 }
 
-#' Drop duplicate rows from this DataFrame
```

Comment:
Should have been "LazyFrame", but I removed it so that we can inherit docs in eager